### PR TITLE
[inductor] Tile after fusion

### PR DESCRIPTION
**Pull request by jansel**

### CODE REVIEW

File: torchinductor/triton_ops/autotune.py [hunk 1]
```diff
@@ -147,13 +176,28 @@ def pointwise_heuristics(size_hints):
...
             ],
             key=["xnumel", "ynumel"],
         )
     if len(size_hints) == 3:
-        return apply_triton_config(triton_config(size_hints, 16, 16, 16))
+        if not config.triton.autotune:
+            return apply_triton_config(triton_config(size_hints, 16, 16, 16))
+        return autotune(
+            [
+                triton_config(size_hints, 16, 16, 16),
```
Comment (Chillee) [inline torchinductor/triton_ops/autotune.py:191]: these look pretty suspicious lol.

Comment (jansel) [reply torchinductor/triton_ops/autotune.py:191]: What would you suggest?

Comment (ngimel) [reply torchinductor/triton_ops/autotune.py:191]: Getting size_hints == 3 should be pretty unusual, do we know under which conditions we are getting it?

Comment (jansel) [reply torchinductor/triton_ops/autotune.py:191]: `config.max_tiles=2` right now, so never.

I was experimenting a bit with `max_tiles=3` (3D) tiling, for the case where you read from 3 different tensors each with different stride=1 dimensions

File: torchinductor/triton_ops/autotune.py [hunk 2]
```diff
@@ -147,13 +176,28 @@ def pointwise_heuristics(size_hints):
         return autotune(
             [
                 triton_config(size_hints, 64, 64),
```
Comment (ngimel) [inline torchinductor/triton_ops/autotune.py:178]: replace with 32 x 32?

**Review by ngimel — APPROVED** [review]

File: torchinductor/triton_ops/autotune.py (hunk 2 above)
Comment (jansel) [reply torchinductor/triton_ops/autotune.py:178]: will fix in next pr, merging to unblock @pyjhzwh

**Pull request merged**